CMakeLists.txt

```diff
@@ -196,7 +196,8 @@ target_include_directories(ccpp_physics PUBLIC
 target_link_libraries(ccpp_physics PRIVATE MPI::MPI_Fortran)
 target_link_libraries(ccpp_physics PUBLIC w3emc::w3emc_d
                                           sp::sp_d
-                                          NetCDF::NetCDF_Fortran)
+                                          NetCDF::NetCDF_Fortran
+				          fms)
```

Comment:
Yeah, this should not be added by default. What we could do is set a cmake variable in the calling cmakelists.txt file that we can test. For example, we do this in NEPTUNE:
````
set(NEPTUNE_BUILD ON CACHE BOOL "Enable build as part of NEPTUNE" FORCE)
```
We could do something similar for FV3, or we can agree on a variable like `CCPP_HOST_MODEL=...` and then query if it is `NEPTUNE`, `FV3`, ...